File: scala-gatling-client/src/gatling/scala/io/swagger/client/model/UserObject.scala

package io.swagger.client.model

import java.util.HashMap

case class UserObject (
    _id: String,
    _username: String,
    __meta: _metaObject
)
object UserObject {
    def toStringBody(var_id: Object, var_username: Object, var__meta: Object) =
        s"""
        | {
        | "id":$var_id,"username":$var_username,"_meta":$var__meta
        | }
        """.stripMargin
}

File: dart-jaguar-client/lib/model/user_object.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:swagger/model/meta_object.dart';
part 'user_object.jser.dart';

class UserObject {
  
  final String id;
  
  final String username;
  
  final MetaObject meta;
  

  UserObject(
    

{
    
     this.id = null,  
     this.username = null,  
     this.meta = null 
    }
  );

  @override
  String toString() {
    return 'UserObject[id=$id, username=$username, meta=$meta, ]';
  }
}

@GenSerializer()
class UserObjectSerializer extends Serializer<UserObject> with _$UserObjectSerializer {

}
